openreview/conference/templates/programchairWebfield.js

```diff
@@ -266,6 +273,49 @@ var postReviewerEmails = function(postData) {
   });
 };
 
+var getReviewerBidCounts = function() {
+  if (!BID_NAME) {
+    return $.Deferred().resolve({});
+  }
+
+  return Webfield.getAll('/edges', {
+    invitation: REVIEWERS_ID + '/-/' + BID_NAME,
```

Comment:
area chairs have a different invitation to get the edges, it starts with the AREA_CHAIRS_ID

Reply:
Ah ok, right now I am not showing the area chair bids, but I could get those here as well